build.backend.sh

```
@@ -10,7 +10,9 @@ else
     FLAVOR=""
 fi
 
-export SDKROOT=$(xcrun --sdk macosx --show-sdk-path)
+if [[ $(uname) == "Darwin" ]]; then
+    export SDKROOT=$(xcrun --sdk macosx --show-sdk-path)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)